main.go

```diff
@@ -138,6 +139,19 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+func CreateStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandle, err error) {
+	var tokenSrc oauth2.TokenSource
+
+	tokenSrc, err = auth.GetTokenSource(context.Background(), flags.KeyFile, flags.TokenUrl, true)
+	var storageClientConfigToken = storage.StorageClientConfig{TokenSrc: tokenSrc}
+	if err != nil {
```

Comment:
you should check for err != nil before creating storageClientConfigToken instance

Reply:
fixed